daemon/graphdriver/vfs/driver.go

```diff
@@ -30,6 +29,27 @@ func Init(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 		home:      home,
 		idMapping: idtools.NewIDMappingsFromMaps(uidMaps, gidMaps),
 	}
+
+	for _, option := range options {
+		key, val, err := parsers.ParseKeyValueOpt(option)
+		if err != nil {
+			return nil, err
+		}
+		key = strings.ToLower(key)
+		switch key {
+		case "copy.concurrency":
```

Comment:
Wondering if this should this be `vfs.copy-concurrency` (if it's specific to the `vfs` driver) 🤔 @cpuguy83 ?